four.html

```diff
@@ -0,0 +1,27 @@
+<html>
+
+<body>
+<form id="myform">
+    <h2>Add a User:</h2>
+    <input id="username" type="text" name="username" placeholder="name">
+    <input id="email" type="email" name="email" placeholder="email">
+    <button onclick='addUser()' type="submit">add user</button>

```

Comment:
Storage should be used for adding user and should fetched while showing user list